types/stubs/logging/config.pyi

```diff
@@ -28,7 +28,7 @@ class ConvertingDict(dict, ConvertingMixin):
 
 class ConvertingList(list, ConvertingMixin):
     def __getitem__(self, key: Any): ...
-    def pop(self, idx: int = ...): ...
+    def pop(self, idx: int = ...): ... # type: ignore[override]
```

Comment:
```
types/stubs/logging/config.pyi:31: error: Argument 1 of "pop" is incompatible with supertype "list"; supertype defines the argument type as "SupportsIndex"  [override]
types/stubs/logging/config.pyi:31: note: This violates the Liskov substitution principle
types/stubs/logging/config.pyi:31: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides
```
I'm not actually sure we should ignore the override error here:
https://clifford.readthedocs.io/en/v1.3.0/_modules/typing.html
https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides

It seems to me that there's no reason we should be overriding pop here?

Reply:
Maybe my search wasn't accurate but I can't find anywhere ConvertingList is even referenced?

Reply:
Yeah, I've come across things in stubs that are that way. Maybe it's a place Vince used to define interface without actually implementing it?